deploy/mainnet/130_deploy_Multicall.ts

```diff
@@ -17,6 +19,24 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
     skipIfAlreadyDeployed: true,
   })
+
+  const contractOwnerAddress = "0x05f32b3cc3888453ff71b01135b34ff8e41263f2"
+  const impersonatedOwner = await impersonateAccount(contractOwnerAddress)
+  await hre.network.provider.send("hardhat_setBalance", [
+    await impersonatedOwner.getAddress(),
+    `0x${(1e18).toString(16)}`,
+  ])
+  const Multicall3 = await ethers.getContractFactory(
+    "Multicall3",
+    impersonatedOwner,
+  )
+  const multicall3 = await Multicall3.deploy()
+  await multicall3.deployed()
+  await save("Multicall3", {
+    abi: multicall3.abi,
+    address: multicall3.address,
+  })
+  console.log("multicall3 deployed to:", multicall3.address)
```

Comment:
Using `log` function from hardhat-deploy will be better for consistency. It will skip printing on tests and coverage.

Reply:
This portion should also be logic'd off so that this only runs in hardhat local node. 
A way to check this is when running `npm run fork`, it does not attempt to impersonate/deploy.